### ISSUE TITLE
F.lux no supported on Ubuntu 22.04

### ISSUE THREAD

**Issue by alexTitakoff** [xflux-gui/fluxgui#156]
Good afternoon.

I have been using Brightness Controller and F.lux for a long time.
https://github.com/LordAmit/Brightness
https://justgetflux.com/

Installed Ubuntu 22.04. Do not work. And all the programs that are connected with the brightness and RGB interface, as I understand it.

On 16.04 everything is ok
. 20.04 everything is ok. Upgraded the same system to 22.04 - the programs remained, stopped working.
20.04 on another PC (mac mini) everything is ok.

Question. Is it possible to find the problem? Who has encountered?
Who in theory should finalize Canonical or vendors?

**Comment by abejaranoh** [comment]
did you install it manully or via ppa ?

**Comment by alexTitakoff** [comment]
Installed manually. But I didn't understand why I couldn't through the ppa.

That is, it is not being updated at the moment?

I solved the problem with comfortable work for the eyes. I didn't know that Ubuntu 22.04 already has a built-in night mode. I think I have enough of it. In extreme cases, the Brightness Controller has a heat setting.

For Brightness Controller, I disabled Wayland and switched to X.Org , Everything worked.

https://dzen-code.ru/ubuntu-22-04-display-brightness-and-nightshift/
